eng/Version.Details.xml

```diff
@@ -52,7 +52,7 @@
     </Dependency>
     <Dependency Name="Microsoft.Build" Version="16.8.0" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
       <Uri>https://github.com/dotnet/msbuild</Uri>
-      <Sha>126527ff107ae93fed10af675506c56d046aa5a3</Sha>
+      <Sha>c6ed2fef9d1cb5504eb51c6f6a9d529e4efd9054</Sha>
```

Comment:
Probably shouldn't update our 5.0 release branch (source-build `master`) to a 6.0 commit (msbuild `master`). 😛

Reply:
MSBuild master is 16.9 is 5.0.200.

Reply:
Oh, interesting. We do only build 5.0.1xx, so the logic still applies, but thanks for the correction.